### DISCUSSION TITLE
Mis practicas.

### DISCUSSION THREAD

**Discussion by Cerecero** [Launch-X-Latam/MisionFrontEnd#400] [Show and tell]
Hola compañeros, aqui dejo mis practicas finalizadas :smiley:. Saludos.
|Practica|Links|
|---|---|
|Toma de Requerimientos|[Abogabot](https://github.com/Cerecero/Abogabot)|
|HTML|[Github Page](https://github.com/Cerecero/ice-cream)<br>[Index Page](https://cerecero.github.io/ice-cream/)<br>[Pagina de pedido](https://cerecero.github.io/ice-cream/compras.html#pedido)<br>[Admin Page](https://cerecero.github.io/ice-cream/admin.html)|
|CSS|[Vaccination Github Page](https://github.com/Cerecero/vaccination-mockup)<br>[Index Page](https://cerecero.github.io/vaccination-mockup/)|
|Js|[Pokedex Github Page](https://github.com/Cerecero/pokedex)<br>[Pokedex](https://cerecero.github.io/pokedex/)|

**Comment by safraZX** [comment]
Hola, que cool te quedo todo, mi pregunta es, las debos subir a un repo no, o solo asl dejo aqui en discussion asi como tu ?

**Reply by Cerecero** [reply]
Si la tienes que subir a tu repo, y si quieres ya las puedes compartir con la comunidad.

**Comment by IngGustavo** [comment]
Que tal, te quedo genial.
Una duda, ¿Cómo le haces para poner un link donde te mande a la pagina para verla en el navegador?

**Reply by safraZX** [reply]
puedes usar git hub pages para poder vizualizar la pagina asi no tienes que hostear ya que estaria lit todo en git hub ya
